src/ui/hooks/recordings.ts

```diff
@@ -30,6 +30,9 @@ export function useGetRecording(recordingId: RecordingId) {
           user {
             invited
           }
+          workspace {
```

Comment:
Can I just modify graphQL queries now given the changes to the API?

Reply:
This is fine. The nice thing about graphql is that once we have workspaces supported it should just work

Reply:
Yup!